src/Forms/FormHandler.php

```diff
@@ -524,6 +524,15 @@ public static function GetInstance(): FormHandler
         return self::$Instance;
     }
 
+    public function setDBHandler($DBHandle=null)
```

Comment:
This is already done in the constructor; there's no need to have a method for it. Removed old uses of it in b6bb73a8